backend/src/authentication/api.rs

```diff
@@ -241,6 +241,49 @@ pub async fn development_create_user(
     Ok(StatusCode::CREATED)
 }
 
+/// Development endpoint: login as a user (unauthenticated)
```

Comment:
#1043 removed this endpoint (merged earlier today), I don't think we should be bringing it back in this PR?